### ISSUE TITLE
core: use Newlib as a PKG

### ISSUE THREAD

**Issue by Kijewski** [RIOT-OS/RIOT#1501]
There is a lot of patching around in RIOT because the precompiled Newlib the distributor supplies is not perfectly configured as we need it.

We should think about compiling Newlib ourself. Of course this would mean a little bigger compiling time for the first project that you build after checking out RIOT, but it's only once per board.

**Comment by miri64** [comment]
Since by now every package is downloaded by every application for every board I don't think that's still a good idea...

**Comment by OlegHahm** [comment]
Unless you use git-cache.

**Comment by OlegHahm** [comment]
Close as wontfix?

**Comment by kaspar030** [comment]
Two years old. yes.
